cmd/plugins_test.go

```diff
@@ -313,3 +361,25 @@ func TestLambdaFileWalker(t *testing.T) {
 		}
 	})
 }
+
+func signalHandler(tempRootDir string, cleanup func()) {
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, syscall.SIGINT, syscall.SIGTERM)
+	exitChan := make(chan int)
+	go func() {
+		s := <-signalChan
+		test.FuncEnter()
+		defer test.FuncExit()
+		switch s {
+
+		case syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT:
+			fmt.Println("Received an ", s.String(), "cleaning ", tempRootDir, "and exiting")
+			cleanup()
+			os.Exit(0)
+
+		default:
```

Comment:
Is this case ever hit? If not sure, just remove.

Reply:
It is just a defensive coding catch all so we can know if some unknown signal was received.